sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/TextDocumentInput.java

```diff
@@ -82,6 +68,17 @@ public String getLanguage() {
         return this.language;
     }
 
+    /**
+     * Set the language property: (Optional) This is the 2 letter ISO 639-1
+     * representation of a language. For example, use "en" for English; "es"
+     * for Spanish etc. If not set, use "en" for English as default.
+     *
+     * @param language Optional. This is the 2 letter ISO 639-1 representation of a language.
+     */
+    public void setLanguage(String language) {
```

Comment:
Should follow Fluent pattern.

Reply:
not sure why fluent pattern. ID and text are required.